hy/core/shadow.hy

```diff
@@ -138,11 +138,17 @@
 (defn not-in [x y]
   (not-in x y))
 
+(defn get [thing key1 &rest keys]
+  (for* [k (+ (, key1) keys)]
+    (setv thing (get thing k)))
+  thing)
```

Comment:
I don't think this kind of micro-optimization is worth grubbying up the code. I'll use a leading `setv` instead of tuple addition, but I would only support adding an `if` for a performance rationale if I had good reason to believe that this function is going to be a CPU bottleneck in real code and the `if` makes a meaningful difference.